docs/troubleshooting.md

```diff
@@ -0,0 +1,141 @@
+wavefront   Healthy   Running (1/1)   Running (1/1)       Running (3/3)    Running (3/3)   3h3m   All components are healthy
+```
+Verify that the `STATUS` column is `Healthy` and all the requested resources are running i.e., `Running (1/1)`.
+
+### Is there a proxy backlog?
+
+You can check if the proxy is having backlog issues by following the instructions on [406 - Cannot Post Push Data WARN Message](https://docs.wavefront.com/proxies_troubleshooting.html#proxy-warn-messages). 
+If your proxy is having backlog issues, below are some options for fixes:
+- Filter more metrics - Refer [this example scenario](https://github.com/wavefrontHQ/wavefront-operator-for-kubernetes/blob/main/deploy/kubernetes/scenarios/wavefront-collector-filtering.yaml) for filtering metrics
+- Increase limits - Contact VMware Tanzu Observability Support to request a higher backend limit as suggested in [406 - Cannot Post Push Data WARN Message](https://docs.wavefront.com/proxies_troubleshooting.html#proxy-warn-messages)
+
+### Are metrics being dropped?
+
+Check the proxy logs by running the below command.
+```
+kubectl logs deployment/wavefront-proxy -n observability-system
+```
+If you see the error `Too many point tags` in the proxy logs, below is an option to fix the error:
+- Drop tags - Use `tagExclude` to drop tags. Refer [here](https://github.com/wavefrontHQ/wavefront-operator-for-kubernetes/blob/main/deploy/kubernetes/scenarios/wavefront-full-config.yaml) for an example scenario
+
+### Are Metrics being filtered?
+
+Check custom resource configuration for verifying the metrics that are being filtered, by running the below command.
+```
+kubectl describe wavefront -n observability-system
+```
+If you would like to customize the metrics being filtered, refer [here](https://github.com/wavefrontHQ/wavefront-operator-for-kubernetes/blob/main/deploy/kubernetes/scenarios/wavefront-collector-filtering.yaml) for an example scenario.
+
+### Is my Custom Resource Config File Configured Correctly
+
+Check for unhealthy status by running below command.
+```
+kubectl get wavefront -n observability-system
+```
+If there are any configuration or validation errors, the `MESSAGE` column in the result will describe the error.
+
+## Running workloads not being discovered or monitored
+
+- Check the Wavefront Collector Troubleshooting dashboard in the Kubernetes integration for collection errors. The `Collection Errors per Type` chart and `Collection Errors per Endpoint` chart can be used to find the sources whose metrics are not being collected
+- Refer to [this](https://github.com/wavefrontHQ/wavefront-operator-for-kubernetes/blob/main/deploy/kubernetes/scenarios/wavefront-full-config.yaml) example scenario for configuring sources for metric collection
+- Check the cluster collector logs to verify if the source was configured for the metrics to be collected
+```
+kubectl logs deployment/wavefront-cluster-collector -n observability-system
+```
```

Comment:
I am inclined to remove the last point on using `kubectl logs deployment/wavefront-cluster-collector -n observability-system` to find the sources being scraped, because I found that cluster collector does not log all the sources, like the [telegraf_mysql_plugin_source](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes/blob/main/hack/test/deploy/mysql-config.yaml) (with `namespace: observability-system`). 
The node collector where the mysql pod ends up being, is what is logging the collection of metrics from `telegraf_mysql_plugin_source`. But unfortunately I could not find a way to get the logs of all pods from a daemonset (Using `kubectl logs daemonset/wavefront-node-collector -n observability-system` prints log of the cluster collector pod :confused:)
So unless we figure out how to get logs of all daemonset node collector pods, we should consider removing the last point in the list maybe?